src/com/google/javascript/jscomp/CommandLineRunner.java

```diff
@@ -1064,6 +1063,7 @@ private void initConfigFromFlags(String[] args, PrintStream out, PrintStream err
       // For backwards compatibility, allow both commonJsPathPrefix and jsModuleRoot.
       List<String> moduleRoots = new ArrayList<>(flags.commonJsPathPrefix);
       moduleRoots.addAll(flags.moduleRoot);
+      moduleRoots.add(ES6ModuleLoader.DEFAULT_FILENAME_PREFIX);
```

Comment:
I'm not sure if that's correct.

Imagine you compile files `a/b.js` and `./c.js`. `ES6ModuleLoader` will, if given `./` as a module root, find `a/b` and `c` as modules. If you don't have `./` as a root, it will not normalize the `./c.js` and won't find it if code just write `import {something} from 'c';`.

So IMHO we do need `./`, and we should maybe actually think about also adding the absolute URI of the current working directory by default. E.g. if somebody specifies their inputs with absolute paths like `/home/user/foo.js` and `/home/user/bar.js`, then runs the compiler in `/home/user`, we maybe should find just `import {x} from 'bar';` relative to `foo.js`.

But let's fix that if/when somebody actually has the problem.